src/pid.rs

```diff
@@ -87,6 +87,12 @@ impl Pid {
     }
 }
 
+impl fmt::Display for Pid {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{}", self.0)
```

Comment:
```suggestion
        self.0.fmt(f)
```

This makes some of the custom alignments etc. work.